x-pack/metricbeat/module/mssql/performance/performance.go

```diff
@@ -65,6 +65,9 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 func (m *MetricSet) Fetch(reporter mb.ReporterV2) {
 	var err error
 	var rows *sql.Rows
+	var buffer_cache_hit_ratio, buffer_cache_hit_ratio_base int64
```

Comment:
One nit: It's not conventional to use underscores to for multi-word names in. Go, See [this](https://go.dev/doc/effective_go#mixed-caps). 
```suggestion
	var bufferCacheHitRatio, bufferCacheHitRatioBase int64
```

Reply:
same suggested name, used for string purpose.

Reply:
I noticed that, but I'd still suggest to rename it to something else, but in mixed caps.

Reply:
Something like this might make more sense.

```go
bufferCacheHitRatioColumn := "Buffer cache hit ratio"
bufferCacheHitRatioBaseColumn := "Buffer cache hit ratio base"
var bufferCacheHitRatioValue, bufferCacheHitRatioBaseValue int64
```